currencies/src/lib.rs

```diff
@@ -93,10 +93,13 @@ const RADIANS_PER_DEGREE: U0F64 = U0F64::from_bits(0x0477D1A894A74E40);
 // in meters
 const MEAN_EARTH_RADIUS: I32F0 = I32F0::from_bits(0x006136B8);
 
+const INITIAL_NEWBIE_TICKETS: u8 = 50;
+
 decl_storage! {
     trait Store for Module<T: Trait> as EncointerCurrencies {
         Locations get(fn locations): map hasher(blake2_128_concat) CurrencyIdentifier => Vec<Location>;
         Bootstrappers get(fn bootstrappers): map hasher(blake2_128_concat) CurrencyIdentifier => Vec<T::AccountId>;
+        BootstrapperNewbieTickets get(fn bootstrapper_newbie_tickets): double_map hasher(blake2_128_concat) CurrencyIdentifier, hasher(blake2_128_concat) T::AccountId => u8;
```

Comment:
this should be with the ceremonies pallet, not currencies. reasoning: Cantillon has `currencies` onchain and `ceremonies` off-chain in STF